cfgov/jinja2/v1/_includes/molecules/related-topics.html

```diff
@@ -21,7 +27,9 @@ <h2 class="header-slug">
             {{ value.heading or 'Related Topics' }}
         </span>
     </h2>
-    <ul class="list list__unstyled list__links">
+    <ul class="list
+               list__unstyled
+               list__links">
```

Comment:
I tend to when it gets to be three or more with multiple words. I don't have a strong preference though.